SonarQube.Bootstrapper/ArgumentProcessor.cs

```diff
@@ -200,14 +242,16 @@ private static IBootstrapperSettings CreatePostProcessorSettings(IList<string> b
         /// command line arguments
         /// </summary>
         /// <remarks>We don't want to forward these arguments to the pre- or post- processor</remarks>
-        private static List<string> RemoveBootstrapperArgs(string[] commandLineArgs)
+        private static IList<string> RemoveBootstrapperArgs(string[] commandLineArgs)
         {
             if (commandLineArgs == null)
             {
                 throw new ArgumentNullException("param");
             }
 
-            return commandLineArgs.Except(new string[] { BeginVerb, EndVerb }).ToList();
+            var excludedVerbs = new string[] { BeginVerb, EndVerb, InstallTargetsVerb };
+
+            return commandLineArgs.Where(arg => !excludedVerbs.Any(e => arg.StartsWith(e, StringComparison.Ordinal))).ToList();
```

Comment:
The previously code was only intended to strip out values that were complete matches e.g. "begin" would be stripped, but "beginxxx" would not. Your version will incorrectly strip out e.g. "beginxxx".

The terms "verb" and "prefix" are not defined anywhere, but "verb" was used informally to refer to arguments that were complete on their own ("begin", "end"). "Prefix" was used to refer to the identifier for an argument that a value part too e.g. /k:name, /d:key=value, so "/i:" is prefix, rather than a verb.